// Copyright 2023 Google. This software is provided as-is, without warranty or representation for any use or purpose.
// Your use of it is subject to your agreement with Google.
 
syntax = "proto2";
 
message SampleData {
  optional int64 window_start = 1;
  optional int64 window_end = 2; 
  optional string key = 3;
  optional int64 total = 4; 
  optional int64 count = 5; 
  optional float avg = 6;
  optional string _CHANGE_TYPE=7;
}